src/api/actions/uploadInBackground.ts

```diff
@@ -0,0 +1,147 @@
+import { checkBulkUploadJob } from "./checkBulkUploadJob";
+import { startBulkUpload } from "./startBulkUpload";
+import {
+  JobIdToUrl,
+  UploadJobStatus,
+  UrlToIPFS,
+  UrlToJobId,
+} from "../../types";
+import * as fs from "fs";
+
+const URLS_PER_CHUNK_START = 100;
+const URLS_PER_CHUNK_CHECK = 1000;
+const ERROR_FILEPATH = "error.json";
+const FAILURE_FILEPATH = "failed.json";
+
+export interface StartErrorJob {
+  url: string;
+  error?: unknown;
+}
+
+export interface CheckErrorJob {
+  url: string;
+  jobId: string;
+  error?: unknown;
+}
+
+let errorJobs: {
```

Comment:
Don't use global/static variables like this. If i run this function twice without killing my node process, it will re-use the old data.

Reply:
If you use global/static variables to maintain a cache or something, that's great, but in this case the data is only relevant for a single run of `uploadInBackground`

Reply:
If you need these variables in another function, consider passing them, or declaring those functions in-line so it can reference capture them